data/human/human missions.txt

```diff
@@ -4927,3 +4927,75 @@ mission "Riley"
 			label end
 			`	You say your goodbyes, leaving to your ship and returning to the stars.`
 				decline
+
+
+
+mission "Wool Smuggling 1"
+	minor
+	name "Pickup at <planet>"
+	description "Head to <destination> to pick up a shipment of <tons> of woollen goods for delivery to Bloodsea."
+	cargo "(reserved)" 15
```

Comment:
Would it not be easier to put a block on the next mission and have this one not resolve until that one is offered?
That way people could still carry extra trade goods to New Kansas if they wanted to.

Reply:
That doesn't work; mission completion resolves before offering, and doesn't trigger again until you land/reload the save.

Reply:
So just have the next mission trigger on New Kansas if this one has `offered`.
Have the second mission give you a blocked message that you can't take it without clearing cargo - potentially have it make you visit the spaceport so you can repeat test it without taking off and landing again.
Then have accepting the 2nd mission fail the first.
The reward is on the second one after all.